### PR TITLE
170

### PR DESCRIPTION
**Pull request by harpatel1**
Fix to 500 when creating concept description.

### CODE REVIEW

File: ocl/concepts/tests.py [hunk 1]
```diff
@@ -243,6 +243,41 @@ def test_concept_display_name_preferred(self):
...
+        concept = Concept(
+            mnemonic='concept1',
+            created_by=self.user1,
+            updated_by=self.user1,
+            parent=self.source1,
+            concept_class='Diagnosis',
+            names=[self.name],
+            # descriptions=[self.description],
+            datatype="None"
+        )
```
Comment (rkorytkowski) [inline ocl/concepts/tests.py:259]: To mimic the issue you should save the concept now without descriptions.

File: ocl/concepts/tests.py [hunk 2]
```diff
@@ -243,6 +243,41 @@ def test_concept_display_name_preferred(self):
...
+        description_1 = LocalizedText(
+            name='concept description',
+            locale='en',
+            locale_preferred=True,
+            type='ConceptDescription'
+        )
+
+        concept.descriptions = [description_1]
+        concept.full_clean()
+        concept.save()
```
Comment (rkorytkowski) [inline ocl/concepts/tests.py:270]: To mimic the issue you should be calling concept_version.clone() and  ConceptVersion.persist_clone(new_version, request.user) same as in https://github.com/OpenConceptLab/oclapi/blob/4d37049cdade81a82178af1b2cbd8509343d1544/ocl/concepts/views.py#L441-L446

**Comment by rkorytkowski** [comment]
Looks good. Please squash into a single commit. Start the commit message from: OpenConceptLab/ocl_issues#170

**Comment by harpatel1** [comment]
All commit messages are already starting with 170. Also why can't we have multiple commits merged ?

**Comment by rkorytkowski** [comment]
We need the commit message to mention "OpenConceptLab/ocl_issues#170" exactly so that GH matches it with the correct repo and issue. I don't mind having multiple commits for larger chunks of work that have long lifespan. Otherwise single commits are easier to cherry-pick, revert and inspect.

**Comment by harpatel1** [comment]
ok, make sense. I'll update it PR.

**Pull request merged**